bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/TransparencyColorImageGcDrawer.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Yatta and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Yatta - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.internal;
+
+import org.eclipse.swt.graphics.*;
+
+public abstract class TransparencyColorImageGcDrawer implements ImageGcDrawer {
+
+	private final Color transparencyColor;
+
+	public TransparencyColorImageGcDrawer(Color transparancyColor) {
```

Comment:
Still one leftover :-) 
```suggestion
	public TransparencyColorImageGcDrawer(Color transparencyColor) {
```